synapseAnnotations/data/common/minimal_Sage_standard.json

```diff
@@ -1045,6 +1055,41 @@
         "value": "CD8+ T-Cells",
         "description": "",
         "source": ""
+      },
+      {
+        "value": "CRISPR arachnoid",
+        "description": "",
+        "source": ""
+      },
+      {
+        "value": "primary arachnoid",
```

Comment:
Is this to distinguish between cultured? If so both cells are arachnoid and isCellLine is altered...